bugtool/cmd/configuration.go

```diff
@@ -214,6 +218,11 @@ func defaultCommands(confDir string, cmdDir string, k8sPods []string) []string {
 		commands = append(commands, tcCommands...)
 	}
 
+	// We want to collect this twice: at the very beginning and at the
+	// very end of the bugtool collection, to see if the counters are
+	// increasing.
+	commands = append(commands, "cat -u /proc/net/xfrm_stat")
```

Comment:
Out of curiosity, why are we using `cat -u` here and not in the other commands?

Reply:
Ouch, that's a good question which I should be documenting in the comment :disappointed: 

The output from all these commands is written into bugtool files whose name is generated. More specifically, the filenames are generated from the commands themselves. So for example this command's output will be stored in `cat--u-proc-net-xfrm_stat.md` IIRC. Hence, we need some difference in the command to end up with two difference filenames. I picked `-u` because that has no effect.

Reply:
Thanks a lot for elaborating and documenting it in a comment!